x-pack/plugin/esql/qa/testFixtures/src/main/resources/where-like.csv-spec

```diff
@@ -319,3 +319,63 @@ warningRegex:java.lang.IllegalArgumentException: single-value function encounter
 emp_no:integer | job_positions:keyword 
 10025          | Accountant 
 ;
+
+likeWithUpper
+FROM employees
+| KEEP emp_no, first_name
+| SORT emp_no
+| WHERE TO_UPPER(first_name) LIKE "GEOR*"
+;
+
+emp_no:integer |first_name:keyword
+10001          |Georgi
+10055          |Georgy
+;
+
+likeWithLower
```

Comment:
If we decide to accept the breaking change, you'll need at least a new capability, otherwise these tests will fail on mixed clusters.

Reply:
No longer needed.